{"name": "Проверка попытки логина курьера без поля login", "status": "passed", "description": "Пытаемся залогиниться на аккаунт без поля Логин и получаем ошибку 400 и соответствующий текст ответа", "start": 1721375125394, "stop": 1721375125514, "uuid": "86334e9d-cc3c-46db-a4cf-8ce3ff3f0a66", "historyId": "c9626acb2d8f298ffc51c9dc636b43e9", "testCaseId": "c9626acb2d8f298ffc51c9dc636b43e9", "fullName": "tests.test_courier_login.TestLoginCourier#test_not_login_courier", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "LAPTOP-NE0EJ4CA"}, {"name": "thread", "value": "10612-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}